service/controller/v18/version_bundle.go

```diff
@@ -36,6 +46,6 @@ func VersionBundle() versionbundle.Bundle {
 			},
 		},
 		Name:    "kvm-operator",
-		Version: "3.2.1",
+		Version: "3.2.0",
```

Comment:
Why is that a downgrade? Should at least be a minor bump IMO.

Reply:
the previous version is `3.1.1`. I don't know why it got `3.2.1` and not `3.2.0`.